tests/generator/generate.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/python3
```

Comment:
I don't think this file needs a shebang or is particularly well named. base.py maybe?

Reply:
Copy-pasta fallout. Thanks!